content/en/docs/contributing/coding-conventions.md

```diff
@@ -50,3 +50,15 @@ import (
 
 Once this manual split of standard library, external and internal imports has been made, it will be
 enforced automatically by `goimports` when executed in the future.
+
+## UK vs. US spelling
+
+For the sake of consistency, we settled on using the US spelling for the
```

Comment:
nitpick: I think the first "the" isn't required, and I'd maybe phrase it very slightly differently:

```suggestion
For the sake of consistency, cert-manager uses US spelling for the
```